src/dapla_pseudo/v1/supported_file_format.py

```diff
@@ -10,10 +16,23 @@ class SupportedFileFormat(Enum):
     XML = "xml"
     PARQUET = "parquet"
 
-    def get_pandas_function_name(self) -> str:
-        """Return the pandas function name for the file format."""
+    def get_function_name(self) -> str:
+        """Returns the reader function name for the file format."""
         return f"read_{self.value}"
```

Comment:
I feel like this would be much simpler with a direct map to each function, then there won't be any need for logic to know whether it should be called from Polars or Pandas.

Reply:
That's a good suggestion. It should be fixed in:
- [Map filetypes to their corresponding reader functions](https://github.com/statisticsnorway/dapla-toolbelt-pseudo/pull/244/commits/3b91a446e491f195746756c24ac0a3366b401411)